.github/workflows/doc-tests.yml

```diff
@@ -24,6 +24,12 @@ jobs:
             args: -W
     steps:
       - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+          cache: 'pip'
```

Comment:
this doesn't work as is.

Reply:
What do you mean? I see the cache being hit even before it was merged [^1] (seems it only requires one workflow to run successfully)

[^1]: https://github.com/teemtee/tmt/actions/runs/11161108734/job/31022955950?pr=3241#step:3:17

Reply:
I was just expecting to see "Using cached" instead of "Downloading", but yeah. Now I see a few of those as well. I'm probably mixing things up, but regardless, seems to work. Thanks.